component/authenticator/src/test/java/org/wso2/carbon/identity/application/authenticator/backupcode/BackupCodeAuthenticatorTest.java

```diff
@@ -285,4 +308,81 @@ public Object[][] dataForProcess() {
                         AuthenticatorFlowStatus.INCOMPLETE, false}
         };
     }
+
+    @Test(dataProvider = "initiateAuthenticationRequestWithErrorContextData")
+    public void testInitiateAuthenticationRequestWithErrorContext(String showError, String showErrorOnLoginPage,
+                                                                  Boolean errorContextPresent, int failedAttempts,
+                                                                  int maxAttempts, String lockedReason,
+                                                                  Boolean hasErrorCode, String errorCodeParam,
+                                                                  Boolean hasLockedReason, String lockedReasonParam)
+            throws AuthenticationFailedException, BackupCodeException, UserStoreException, IOException {
+
+        String username = "TEST-DOMAIN/test@gmail.com";
+        Map<String, String> claims = new HashMap<>();
+        claims.put(BACKUP_CODES_CLAIM, HASHED_BACKUP_CODES);
+
+        BackupCodeAuthenticator backupCodeAuthenticator = new BackupCodeAuthenticator();
+        Map<String, String> parameterMap = new HashMap<>();
+        parameterMap.put(FrameworkConstants.SHOW_AUTHFAILURE_RESON_CONFIG, showError);
+        parameterMap.put(FrameworkConstants.SHOW_AUTH_FAILURE_REASON_ON_LOGIN_PAGE_CONF, showErrorOnLoginPage);
+        AuthenticatorConfig authenticatorConfig1 = new AuthenticatorConfig(
+                "backup-code-authenticator", true, parameterMap);
```

Comment:
Refer `backup-code-authenticator` from constant

Reply:
adressed in e518f298d3807b3fb52c908b88d6ee9be525793b